angular.json

```diff
@@ -115,30 +96,6 @@
               "buildTarget": "ng-zorro-antd-doc:build:production"
             }
           }
-        },
-        "serve-ssr": {
-          "builder": "@angular-devkit/build-angular:ssr-dev-server",
-          "options": {
-            "browserTarget": "ng-zorro-antd-doc:build",
-            "serverTarget": "ng-zorro-antd-doc:server"
-          },
-          "configurations": {
-            "production": {
-              "browserTarget": "ng-zorro-antd-doc:production",
-              "serverTarget": "ng-zorro-antd-doc:production"
-            }
-          }
-        },
-        "prerender": {
-          "builder": "@angular-devkit/build-angular:prerender",
-          "options": {
-            "browserTarget": "ng-zorro-antd-doc:build:production",
-            "serverTarget": "ng-zorro-antd-doc:server:production",
-            "routesFile": "scripts/prerender/route-paths.txt"
-          },
-          "configurations": {
-            "production": {}
-          }
         }
```

Comment:
SSR / SSG no longer require separate builds, as the application builder can support them both!